bin/customs_server.js

```diff
@@ -32,4 +32,5 @@ server(config, log)
   })
   .catch(function (err) {
     log.error({op: 'customs.bin.error', err: err})
+    shutdown()
```

Comment:
nit: please add an exit-code argument to `shutdown` and pass it to `process.exit`, then call `shutdown(1)` here.  Otherwise the process will appear to have exited successfully.